### PR TITLE
Changed success message on bank payment

### PR DESCRIPTION
**Pull request by borislavstoychev**
## Screenshots:
# BG
| Before              | After              |
| ------------------- | ------------------ |
| image | image |

## Screenshots:
# EN
| Before              | After              |
| ------------------- | ------------------ |
| image |image |

## Testing

**New environment variables**:

- `env var` : env var details

**New dependencies**:

- `dependency` : dependency details

**New dev dependencies**:

- `dependency` : dependency details
## Please, take a look on EN version too!

### CODE REVIEW

**Comment by imilchev** [comment]
Doesn't this change actually change the message for both card payments and bank transfers? The issue says that the message should be different only for bank transfers

**Comment by imilchev** [comment]
nvm I saw it now. All good!

File: src/components/one-time-donation/steps/Success.tsx [hunk 1]
```diff
@@ -19,11 +21,13 @@ export default function Success({ donationId }: { donationId?: string }) {
       <Grid container rowSpacing={2} justifyContent={'center'} textAlign="center">
         <Grid item xs={12}>
           <Typography variant="h4" fontSize={14}>
-            {t('success.title')}
+            {(payment && t('success.title-bank')) || t('success.title')}
```
Comment (igoychev) [inline src/components/one-time-donation/steps/Success.tsx:24]: If you need to distinguish if payment was by bank, please check if you still have access to form data and use the payment type from there paymentField.value === 'bank'

Comment (borislavstoychev) [reply src/components/one-time-donation/steps/Success.tsx:24]: > If you need to distinguish if payment was by bank, please check if you still have access to form data and use the payment type from there paymentField.value === 'bank'

Yep, we can access the field 
image
so i will do it on this way....

**Pull request merged**